tests/release/pipelines/rh_advisories.go

```diff
@@ -241,7 +241,14 @@ func createADVSReleasePlanAdmission(advsRPAName string, managedFw framework.Fram
 			"product_name":    "test product",
 			"product_stream":  "rhtas-tp1",
 			"product_version": "v1.0",
-			"type":            "RHBA",
+			"type":            "RHSA",
+			"cves":            []map[string]interface{}{
```

Comment:
Ideally it would be in the Release. ReleasePlan is still slightly better than ReleasePlanAdmission, if possible, as that is closer to what a user may do. The cve structure looks nice though - same way as shown here https://konflux.pages.redhat.com/docs/users/releasing/releasing-with-an-advisory.html#_rhsa_release which is perfect

Reply:
Moved it ReleasePlan. Thanks